deploy/kubevirt-operator.operatorgroup.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
I had been trying to figure out why this was an addition, when it is already on master, now I see that you moved it under `deploy/` which makes sense. Can you update this to be a move from `kubevirt-operator.operatorgroup.yaml --> deploy/kubevirt-operator.operatorgroup.yaml` ?

Reply:
It's already kind'a is, look at the:
https://github.com/djzager/kubevirt-olm/pull/7/commits/364868e0f6fefc085ee5380abc689c789123f767